lib/src/signing.rs

```diff
@@ -53,6 +53,11 @@ pub struct Verification {
     /// A display string, if available. For GPG, this will be formatted primary
     /// user ID.
     pub display: Option<String>,
+    /// The name of the backend that provided this verification.
+    /// Is `None` when no backend was found that could read the signature.
+    ///
+    /// Always set by the signer.
+    backend: Option<String>,
```

Comment:
If this is supposed to be set by `Signer`, I don't think `Verification::new()` should have `backend` argument.

I personally don't mind if it's set by the signing backend, and the field is `pub backend: String`.

Reply:
Ah yeah, I recall that I was a bit unsure about this part.

If the `Signer` should set `backend`, wouldn't we need some sort of setter method then?
Or how would we set it from "outside" `Verification`?

@necauqua, @julienvincent, maybe you remember your intentions? (I think the original changes came from you?)